compass/ocean/tests/baroclinic_gyre/forward.py

```diff
@@ -0,0 +1,118 @@
+from compass.model import run_model
+from compass.step import Step
+
+
+class Forward(Step):
+    """
+    A step for performing forward MPAS-Ocean runs as part of
+    the baroclinic gyre test cases.
+
+    Attributes
+    ----------
+    resolution : float
+        The resolution of the test case (m)
+
+    """
+    def __init__(self, test_case, resolution, name='forward', subdir=None,
+                 long=False):
+        """
+        Create a new test case
+
+        Parameters
+        ----------
+        test_case : compass.TestCase
+            The test case this step belongs to
+
+        resolution : float
+            The resolution of the test case (m)
+
+        name : str
+            the name of the test case
+
+        subdir : str, optional
+            the subdirectory for the step.  The default is ``name``
+
+        long : bool, optional
+            Whether to run a long (3-year) simulation to quasi-equilibrium
+        """
+        self.resolution = resolution
+        if resolution >= 1e3:
+            res_name = f'{int(resolution/1e3)}km'
+        else:
+            res_name = f'{int(resolution)}m'
+
+        res_params = {'80km': {'ntasks': 32,
+                               'min_tasks': 3,
+                               'dt': "'01:00:00'",
+                               'btr_dt': "'00:03:00'",
+                               'mom_del4': "5.0e11",
+                               'run_duration': "'0000_03:00:00'"},
+                      '20km': {'ntasks': 384,
+                               'min_tasks': 32,
+                               'dt': "'00:20:00'",
+                               'btr_dt': "'0000_00:00:20'",
+                               'mom_del4': "2.0e10 ",
+                               'run_duration': "'0000_00:01:00'"}}
```

Comment:
```suggestion
                               'run_duration': "'0000_01:00:00'"}}
```
I'm guessing this should be 1 hour (3 time steps) rather than 1 minute (1/20 of a time step, which gets rounded up to 1 time step).